akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -544,7 +548,7 @@ private[akka] class ShardRegion(
     case msg if extractEntityId.isDefinedAt(msg) => deliverMessage(msg, sender())
     case unknownMsg =>
       log.warning(
-        "Message does not have an extractor defined in shard [{}] so it was ignored: {}",
+        "{}: Message does not have an extractor defined in shard so it was ignored: {}",
```

Comment:
But shall we keep with the convention `[{}]`?

Reply:
imho: not, it does not added more readability and those typeNames are usually a kind of well readable constants